CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## Next Unreleased Version
+- Added `convert_names_case` option to `make_federated_schema`.
```

Comment:
Please rebase on master first.

Reply:
Can do!

Reply:
Ugh, Github did something unexpected when I clicked an update button, give me a moment to update